server/lib/src/value.rs

```diff
@@ -900,6 +905,7 @@ pub enum Value {
 
     TotpSecret(String, Totp),
     AuditLogString(Cid, String),
+    ECDHKey(Vec<u8>),
```

Comment:
This should be the validated and deserialised key state, not the DER bytes here.

Reply:
actually I started with that, but then I got into much trouble when I tried to insert something like `EcKey<Public>` in a `SmolSet`, as it was complaining that the key didn't have `Eq` or `PartialEq`. That's why I went for the bytes